src/planner.ts

```diff
@@ -54,10 +52,9 @@ export interface FunctionCall {
 
 export type ContractFunction = (...args: Array<any>) => FunctionCall;
 
-const paramTypeBytes = new RegExp(/^bytes([0-9]*)$/);
-const paramTypeNumber = new RegExp(/^(u?int)([0-9]*)$/);
+export function isDynamicType(param?: ParamType): boolean {
+    if (typeof param === "undefined") return false
```

Comment:
I don't think this does the same thing as the code you're replacing.

Reply:
I mean we can remove this if statement, the issue was that the value passed to that function could be undefined and it didn't account for that. Could judt mark the param as optional and then remove that

Reply:
Never mind, I thought you were deleting a line you're not. I don't think we should ever be calling this when it could be undefined, though.